app/views/unattended/provisioning_templates/provision/preseed_default.erb

```diff
@@ -18,7 +18,10 @@ description: |
   salt_enabled = host_param('salt_master') ? true : false
   os_major = @host.operatingsystem.major.to_i
   squeeze_or_older = (@host.operatingsystem.name == 'Debian' && os_major <= 6)
-  python_package = (@host.operatingsystem.name == 'Ubuntu' && os_major >= 20) ? 'python3' : 'python'
+  python_package = case
+                   when (@host.operatingsystem.name == 'Ubuntu' && os_major < 20) then 'python'
+                   when (@host.operatingsystem.name == 'Debian' && os_major < 12) then 'python'
```

Comment:
Looks like the package isn't available in bullseye either: https://packages.debian.org/buster/python does exist but https://packages.debian.org/bullseye/python doesn't.
```suggestion
                   when (@host.operatingsystem.name == 'Debian' && os_major < 11) then 'python'
```